distributed/scheduler.py

```diff
@@ -4474,7 +4474,7 @@ async def add_worker(
                     result = plugin.add_worker(scheduler=self, worker=address)
                     if inspect.isawaitable(result):
                         await result
-                except Exception as e:
+                except Exception as e:  # pragma: no cover
```

Comment:
I do think we should test failing plugins even though its not top priority it is important for users to know what happens if their plugin errs, e.g. does the cluster come up? Will the exception only be logged? Something undefined?

```suggestion
                except Exception as e:
```